packages/emails/src/templates/OrganizationCreationEmail.tsx

```diff
@@ -68,7 +68,8 @@ export const OrganizationCreationEmail = (
           For personal events we recommend creating a new account with a personal email address.
           <br />
           <br />
-          Enjoy your new clean link: <a href={`https://${newLink}`}>{newLinkWithoutProtocol}</a>
+          Enjoy your new clean link:{" "}
+          <a href={`${newLink}?orgRedirection=true` ?? ""}>{newLinkWithoutProtocol}</a>
```

Comment:
So that, from the link in email the link shows up as working(and not unpublished)

Reply:
NIT: The "??" does nothing here right?

Reply:
Yeah here it's redundant. I added ?orgRedirection=true later so missed this.